services/security/keyvault/key/key.go

```diff
@@ -257,6 +257,8 @@ func getMOCAlgorithm(algo keyvault.JSONWebKeyEncryptionAlgorithm) (wssdcloudcomm
 		return wssdcloudcommon.Algorithm_RSAOAEP256, nil
 	case keyvault.A256KW:
```

Comment:
Are we still supporting this?

Reply:
No, we'll reject the request for this.

Reply:
Is this used by someother api call, if not we can remove it